### PR TITLE
Rfs1052

### PR DESCRIPTION
**Pull request by kelvinator07**
- Added requirements to start powpeg-node in README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,17 +76,108 @@ sha256sum: WARNING: 19 lines are improperly formatted
...
+    ]
+}
 ```
 
+9. Steps to import and configure the project.
+
 *To import the project to [IntelliJ IDEA](https://www.jetbrains.com/idea/download):*
 
-You can import the project to your IDE. For example, [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/). To import go to `File | New | Project from existing sources...` Select `federate-node/build.gradle` and import. After building, run `co.rsk.federate.FederateRunner`
+You can import the project to your IDE. For example, [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/). To import go to `File | New | Project from existing sources...` Select `federate-node/build.gradle` and import.
```
Comment (marcos-iov) [inline README.md:127]:
```suggestion
You can import the project to your IDE. For example, [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/). To import go to `File | New | Project from existing sources...` Select `powpeg-node/build.gradle` and import.
```

File: README.md [hunk 2]
```diff
@@ -72,17 +76,108 @@ sha256sum: WARNING: 19 lines are improperly formatted
...
+
+- Create a New Application configuration with name **FedRunner**
+
+**VM options**
+
+- -Drsk.conf.file=/<PATH-TO-CONF-FILE>/regtest-fed.conf
+- -Dlogback.configurationFile=/<PATH-TO-LOG-FILE>/logback.xml
+- -Dblockchain.config.hardforkActivationHeights.iris300=0
```
Comment (marcos-iov) [inline README.md:135]: Should we keep this?

File: README.md [hunk 3]
```diff
@@ -72,17 +76,108 @@ sha256sum: WARNING: 19 lines are improperly formatted
...
+
+**Program Arguments**
+
+- --regtest --reset
```
Comment (marcos-iov) [inline README.md:139]:
```suggestion
- --regtest --reset

*Remove `--reset` if you want to keep your db after restarting the node*
```

File: README.md [hunk 4]
```diff
@@ -72,17 +76,108 @@ sha256sum: WARNING: 19 lines are improperly formatted
...
+
+**Module**
+
+- -cp federate-node.main
+
+**Main Class**
+
+- co.rsk.federate.FederateRunner
+
+Then clean and build project using **./gradle clean build**
```
Comment (marcos-iov) [inline README.md:149]:
```suggestion
Then clean and build project using **./gradlew clean build**
```

File: README.md [hunk 5]
```diff
@@ -72,17 +76,108 @@ sha256sum: WARNING: 19 lines are improperly formatted
...
+          path = "A/PATH/TO/YOUR/RSK-KEY.key"
+       }
+       MST {
+          type = "keyFile"
+          path = "A/PATH/TO/YOUR/MST-KEY.key"
+       }
+    }
+    # peers for the bitcoin network
+    bitcoinPeerAddresses = [
+        "127.0.0.1:18444"
```
[the remark this answers is not in the document]
Comment (marcos-iov) [reply README.md:118]: Also, according to the instructions below the default port should be `18332`

File: README.md [hunk 6]
```diff
@@ -72,17 +76,108 @@ sha256sum: WARNING: 19 lines are improperly formatted
...
+   rm -rf bitcoin-0.18.0*
+   As a validation, you can run *bitcoind -daemon*. Run *bitcoin-cli stop* afterwards.
+```
+Create the scripts below as a file and its content:
+
+**bitcoin-node.sh**
+```bash
+#!/bin/bash
+
+bitcoind -regtest -printtoconsole -server -rpcuser=rsk -rpcpassword=rsk -rpcport=18332 -txindex -debug=net --deprecatedrpc=signrawtransaction -addresstype=legacy -deprecatedrpc=accounts -deprecatedrpc=generate
```
Comment (marcos-iov) [inline README.md:169]:
```suggestion
bitcoind -regtest -printtoconsole -server -rpcuser=rsk -rpcpassword=rsk -rpcport=18332 -txindex -debug=net -deprecatedrpc=signrawtransaction -addresstype=legacy -deprecatedrpc=accounts -deprecatedrpc=generate
```

Comment (marcos-iov) [reply README.md:169]: Maybe we could add also `-datadir=PATH/TO/DATA/DIRECTORY`

Not sure if it's required or not

File: README.md [hunk 7]
```diff
@@ -110,10 +205,19 @@ includeBuild('<PATH-TO-RSKJ-SOURCE-CODE>') {
 
 **Note:** Change PATH-TO-RSKJ-SOURCE-CODE value to your local Rskj path.
 
+
+### Steps to run the project.
+
+*To run from command line:*
+
+```bash
+./gradle run -PmainClass=co.rsk.federate.FederateRunner
```
Comment (marcos-iov) [inline README.md:214]:
```suggestion
./gradlew run -PmainClass=co.rsk.federate.FederateRunner
```

File: README.md (hunk 2 above)
Comment (kelvinator07) [reply README.md:135]: I'm Don't Know, Might Need a Second Opinion

**Pull request closed without merging**